frontend/packages/component-library-styles/mixins.scss

```diff
@@ -1,4 +1,4 @@
-@import 'primitiveColors.scss';
+@use 'colors';
```

Comment:
since we still using couple primitiveColors in this file, worth keeping both colors files (e.g. `@use 'colors'; @use 'primitiveColors'`)